src/util/transactions.ts

```diff
@@ -24,15 +21,16 @@ export const parseBlockchainTransaction = (tx, address: string) => {
  * @returns
  */
 export const parseSpendTx = (newtx, from) => {
+  console.log('parseSpendTx ', newtx);
```

Comment:
this is log intended to stay or was added here as temporary debug? (_this part of a bigger question I have of how we should approach logging, what makes sense to log etc._

Reply:
it is a whoopsie